scripts/vmbuild.rb

```diff
@@ -30,17 +30,28 @@
 
 BUILD_BASE          = Pathname.new("/build")
 GPG_DIR             = Pathname.new("/root/.gnupg")
-CFG_DIR             = Pathname.new(__dir__).join("../config")
+SCRIPT_DIR          = Pathname.new(__dir__)
+CFG_DIR             = SCRIPT_DIR.join("../config")
 FILESHARE_DIR       = BUILD_BASE.join("fileshare")
 REFS_DIR            = BUILD_BASE.join("references")
 IMGFAC_DIR          = BUILD_BASE.join("imagefactory")
 IMGFAC_CONF         = CFG_DIR.join("imagefactory.conf")
 STORAGE_DIR         = BUILD_BASE.join("storage")
+ISOS_DIR            = BUILD_BASE.join("isos")
+ISO_FILE            = ISOS_DIR.join("CentOS-Stream-8-x86_64-20211206-dvd1.iso")
 
 FILE_SERVER         = ENV["BUILD_FILE_SERVER"]             # SSH Server to host files
 FILE_SERVER_ACCOUNT = ENV["BUILD_FILE_SERVER_ACCOUNT"]     # Account to SSH as
 FILE_SERVER_BASE    = Pathname.new(ENV["BUILD_FILE_SERVER_BASE"] || ".") # Subdirectory of Account where to store builds
 
+unless cli_options[:ks_log]
+  # Check that the ISO has kickstart logging disabled
+  if system("#{SCRIPT_DIR.join("../bin/iso_modified.sh")} #{ISO_FILE}")
+    $log.error("The ISO '#{ISO_FILE}' was expected to have kickstart logging disabled, but does not. Run bin/iso_modify.sh on it to disable kickstart logging.")
+    exit 1
```

Comment:
I wonder if we should automatically run the `iso_modify` script and log that we're doing so rather than `exit 1`.

Reply:
Won't that be expensive to continually modify the iso even if it's already modified?

Reply:
Or are you saying to add the check to the beginning of the iso_modify script, and then always call the script if there's an ENV var, thereby checking and updating if not modified?